ci-operator/step-registry/openshift/e2e/vsphere/capi/openshift-e2e-vsphere-capi-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: openshift-e2e-vsphere-capi
+  steps:
+    pre:
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-vsphere
+    - chain: ipi-install
+    - ref: capi-conf-apply-feature-gate
```

Comment:
This executes a script `capi-conf-apply-feature-gate-commands.sh` which applies a new featuregate config to a cluster to enable capi functionality. I am not familiar with the recent featuregate changes, so I don't know whether that is clashing with the changes.
The script is setting `featureSet: TechPreviewNoUpgrade` as you are mentioning.
Shall this be removed from the other providers as well then?

Reply:
Yeah we should get that updated, but, it can only be updated for jobs that are on 4.12 plus. So for now, I wouldn't change the workflows, but for any release job you can see configured that has this, you can update the workflow to use `e2e-openshift-vsphere` and then add copy the env with a `FEATURE_SET` from , for example, https://github.com/openshift/release/blob/7ccdfc397e14a05fd1dd166060f8b07fbdb221d2/ci-operator/config/openshift/cluster-cloud-controller-manager-operator/openshift-cluster-cloud-controller-manager-operator-release-4.12.yaml#L101-L107

Reply:
Addressed in the latest commit.